src/xma/src/xmaplugin/xmaplugin.cpp

```diff
@@ -619,18 +576,8 @@ XmaCUCmdObj xma_plg_schedule_cu_cmd(XmaSession s_handle,
         if (return_code) *return_code = XMA_ERROR;
         return cmd_obj_error;
     }
-    if (kernel_tmp1->regmap_size > 0) {
-        if (regmap_size > kernel_tmp1->regmap_size) {
-            xma_logmsg(XMA_ERROR_LOG, XMAPLUGIN_MOD, "Session id: %d, type: %s. Can not exceed kernel register_map size. Kernel regamp_size: %d, trying to use size: %d", s_handle.session_id, xma_core::get_session_name(s_handle.session_type).c_str(), kernel_tmp1->regmap_size, regmap_size);
-            /*Sarab TODO
-            if (return_code) *return_code = XMA_ERROR;
-            return cmd_obj_error;
-            */
-        }
-    }
 
-    uint8_t *src = (uint8_t*)regmap;
-    
+    uint8_t *src = (uint8_t*)regmap;    
```

Comment:
is this used?

Reply:
yes, used in the code and this map is set by user in the XMA api's